components/pkg-export-helm/src/chartfile.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2018 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use failure::SyncFailure;
+use handlebars::Handlebars;
+use serde_json::Value;
+
+use export_docker::Result;
+
+// Keep the default version in main::cli() in sync with this one
+pub const DEFAULT_VERSION: &'static str = "0.0.1";
+
+// Helm chart file template
+const CHARTFILE: &'static str = include_str!("../defaults/HelmChartFile.hbs");
+
+pub struct ChartFile {
+    json: Value,
```

Comment:
What's the reason for using JSON types directly to store the information, rather than creating full types and serializing them out to JSON separately? Seems like you forego type safety this way and make the code a little harder to reason about.

Reply:
Hmm.. sure. That would also mean new() not having to return Result.